rules-config.json

```diff
@@ -0,0 +1,367 @@
+[
```

Comment:
may be delete it? and @aktsay6 will add yml?

Reply:
I can wait until he adds yml, merge it into this branch and only then merge this pull request.